docs/topics/aad.md

```diff
@@ -142,10 +204,7 @@ If you failed at the login page, you may see following error message
 Invalid resource. The client has requested access to a resource which is not listed in the requested permissions in the client's application registration. Client app ID: {UUID} Resource value from request: {UUID}. Resource app ID: {UUID}. List of valid resources from app registration: {UUID}.
 ```
 This could be caused by `Client Application` not being authorized.
-
-1. Go to Azure Portal, navigate to `Azure Active Directory` -> `App registrations`.
-2. Select the `Client Application`, Navigate to `Settings` -> `Required permissions`
-3. Choose `Add`, select the `Server Application`. In permissions tab, select `Delegated permissions` -> `Access {Server Application}`
+See step 2 in Creating client Application on how to authorize client application. 
```

Comment:
Link back to `Create client Application`

`how to authorize client applications` (missing s)